extensions/ql-vscode/src/vscode-tests/run-integration-tests.ts

```diff
@@ -32,11 +67,10 @@ async function main() {
     ];
 
     for (const integrationTestSuite of integrationTestSuites) {
-      // Download and unzip VS Code if necessary, and run the integration test suite.
-      await runTests(integrationTestSuite);
+      await runTestsWithRetry(integrationTestSuite, 2);
```

Comment:
Does running with 2 tries work reliably?  Anecdotally, the integration tests seem to have a surprisingly high probability of failing in the GitHub Actions environment.

Reply:
I mean, it's a tradeoff between how much pointless work we want to do when a test is actually failing. I'm fine bumping this to 3. I have seen it fail twice in a row exactly once locally, but yeah, the rate of failure seems a little higher on CI.